spikeinterface/sorters/runsorter.py

```diff
@@ -52,85 +77,226 @@ def run_sorter(sorter_name, recording, output_folder=None,
     sorting = SorterClass.get_result_from_folder(output_folder)
     
     if delete_output_folder:
-        raise NotImplementedError
-        # TODO : delete_output_folder
+        shutil.rmtree(output_folder)
     
     return sorting
 
 
+def modify_input_folder(rec_dict, input_folder):
```

Comment:
@samuelgarcia this should also handle lists of paths (see the `relative_to` in base.py)